src/style.css

```
@@ -0,0 +1,67 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: Avenir-Heavy, sans-serif;
+  font-size: 16px;
+}
+
+.content {
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+  padding: 0;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  margin-top: 20px;
+}
+
+.content.small {
```

```suggestion
.small {
```
No need for this combination. You can use the exact  selector that you need here